builtin/functions/keys.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/influxdata/flux/interpreter"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/semantic"
+	. "github.com/influxdata/flux/builtin/inputs"
```

Comment:
I do not like the use of this kind of import as it makes it harder to follow the code to its source.
Can you update the code here to use the `inputs` package name?

Reply:
yep, I thought there might be an objection or two, no problem.